app/src/main/java/com/x8bit/bitwarden/ui/autofill/fido2/manager/Fido2CompletionManagerUnsupportedApiImpl.kt

```diff
@@ -0,0 +1,18 @@
+package com.x8bit.bitwarden.ui.autofill.fido2.manager
+
+import com.x8bit.bitwarden.data.autofill.fido2.model.Fido2CredentialAssertionResult
+import com.x8bit.bitwarden.data.autofill.fido2.model.Fido2RegisterCredentialResult
+
+/**
+ * A no-op implementation of [Fido2CompletionManagerImpl] provided when the build version is below
+ * UPSIDE_DOWN_CAKE (34). These versions do not support [androidx.credentials.CredentialProvider].
+ */
+object Fido2CompletionManagerUnsupportedApiImpl : Fido2CompletionManager {
```

Comment:
💭 in place of the empty `{}` could make the overrides into expressions `override fun foo() = Unit`, it may be preferential to leave as is as this has been the convention since the Java days.